src/com/spotz/utils/Utils.java

```diff
@@ -144,11 +183,16 @@ public static boolean isOnline() {
         return netInfo != null && netInfo.isConnectedOrConnecting();
     }
 
+    /**
+     * Changes the language inside the app
+     * @param context
+     */
 	public static void setCurrentLocale(Context context) {
 		SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
 		String value = prefs.getString("keyLanguage", "1");
 		Locale locale = null;
-		switch(Integer.parseInt(value)){
```

Comment:
why don't we store the locale string in the shared preferences instead of numbers which we have to parse?

String value = prefs.getString("keyLanguage", "en");
locale = new Locale(value);

and thats it.